Client/Frontend/Sync/BraveCore/BraveSyncAPI+Utilities.swift

```diff
@@ -38,6 +40,23 @@ extension BraveSyncAPI {
         Preferences.Chromium.syncEnabled.value = false
     }
     
+    func enableSyncTypes() {
+        guard let syncProfileService =
+                (UIApplication.shared.delegate as? AppDelegate)?.braveCore?.syncProfileService else {
+            return
+        }
+
+        syncProfileService.userSelectedTypes = []
+        
+        if Preferences.Chromium.syncBookmarksEnabled.value {
+            syncProfileService.userSelectedTypes.update(with: .BOOKMARKS)
+        }
+        
+        if Preferences.Chromium.syncHistoryEnabled.value {
+            syncProfileService.userSelectedTypes.update(with: .HISTORY)
```

Comment:
How come these enum cases are all-caps?

Reply:
`BraveSyncUserSelectableTypes` defined in brave_sync_profile in brave_core and I guess we didnt think about swift conversion at that point.